src/bit-systems/audio-target-system.ts

```diff
@@ -0,0 +1,209 @@
+import { AComponent, AElement } from "aframe";
+import { addComponent, defineQuery, enterQuery, exitQuery } from "bitecs";
+import { Vector3, Object3D, LineSegments, WireframeGeometry, SphereBufferGeometry, PositionalAudio } from "three";
+import { HubsWorld } from "../app";
+import { AudioSettingsChanged, AudioSource, AudioTarget } from "../bit-components";
+import { SourceType } from "../components/audio-params";
+import { getMediaStream } from "../components/avatar-audio-source";
+import { AUDIO_SOURCE_FLAGS } from "../inflators/audio-source";
+import { AudioSystem } from "../systems/audio-system";
+import { Emitter2Audio, makeAudioEntity } from "./audio-emitter-system";
+
+const createWhiteNoise = (audioContext: AudioContext, gain: number): AudioBufferSourceNode => {
+  const bufferSize = 2 * audioContext.sampleRate,
+    noiseBuffer = audioContext.createBuffer(1, bufferSize, audioContext.sampleRate),
+    gainFilter = noiseBuffer.getChannelData(0);
+  for (let i = 0; i < bufferSize; i++) {
+    gainFilter[i] = (Math.random() * 2 - 1) * gain;
+  }
+
+  const whiteNoise = audioContext.createBufferSource();
+  whiteNoise.buffer = noiseBuffer;
+  whiteNoise.loop = true;
+  whiteNoise.start(0);
+
+  return whiteNoise;
+};
+
+const addSourceToAudioTarget = (audioSourceEid: number, source: AudioNode) => {
+  const audioTargetEids = source2Target.get(audioSourceEid);
+  audioTargetEids?.forEach(audioTargetEid => {
+    const audioEid = Emitter2Audio.get(audioTargetEid)!;
+    const targetAudio = APP.audios.get(audioEid)!;
+    try {
+      targetAudio.disconnect();
+    } catch (e) {}
```

Comment:
May want to leave a comment explaining why this try/catch is necessary (and why we don't log the error)